make_queue/models/models.py

```diff
@@ -115,6 +116,26 @@ class Status(models.TextChoices):
 
     STATUS_CHOICES_DICT = dict(Status.choices)
 
+    status = models.CharField(
+        max_length=2,
+        choices=Status.choices,
+        verbose_name=_("Status"),
+        default=Status.AVAILABLE,
+    )
+    stream_name = models.CharField(
+        max_length=30,
+        verbose_name=_("Stream Name"),
+        default=None, blank=True, null=True,
+        validators=[
+            RegexValidator(
+                regex=slug_re,
+                message=_("Stream name can only consist of english letters, numbers, hyphens or underscores."),
+                code='not_url_safe',
+            )
+        ]
```

Comment:
Some comments on these field options:
* The max length does in principle not need to be constrained, but since it would (most likely) have to be manually copied, we could increase `max_length` to e.g. 50, which is not unwieldy long when manually copying, but still allows for some longer names 🤔
* `null` should generally not be set to `True` on a string-based field (see https://docs.djangoproject.com/en/3.2/ref/models/fields/#null)
* Related to the point above, `default` should not be set to `None`, but the empty string instead
* `verbose_name` should start with a lowercase letter (see https://docs.djangoproject.com/en/3.2/topics/db/models/#verbose-field-names) 😛
* The `RegexValidator` should be moved to `validators.py`, and the `slug_re` regex should be replaced with a regex that disallows capital letters, as allowing both upper- and lowercase letters in a stream name has the potential for unnecessary confusion (as we've previously discussed 😅)

Reply:
It's fine that it's not backwards compatible; we'll just change the RPis to use lowercase at the same time as we deploy 🤠🤠